toolkit/geoview-compose/src/main/java/com/arcgismaps/toolkit/geoviewcompose/OverviewMap.kt

```diff
@@ -0,0 +1,202 @@
+    arcGISMap: ArcGISMap = remember { ArcGISMap(BasemapStyle.ArcGISLightGray) }
+) {
+    OverviewMapImpl(
+        viewpoint = viewpoint,
+        arcGISMap = arcGISMap,
+        modifier = modifier,
+        visibleArea = visibleArea,
+        symbol = symbol,
+        scaleFactor = scaleFactor,
+    )
+}
+
+/**
+ * OverviewMap is a small, secondary [MapView] which shows a representation of the main [GeoView]'s
+ * current [Viewpoint].
+ *
+ * Choose this overload if your main view is a [SceneView]. If a non-default symbol is provided the
+ * symbol must be suitable for a point geometry such as a [SimpleMarkerSymbol].
+ *
+ * @param viewpoint the viewpoint of the main view, this should be provided as a
+ * [ViewpointType.CenterAndScale]
+ * @param modifier the modifier to apply
+ * @param symbol the symbol to apply. Must be suitable for a point geometry.
+ * @param scaleFactor the factor to multiply the main view's scale by. The OverviewMap will
+ * display at the product of mainGeoViewScale * scaleFactor.
+ * @param arcGISMap the map to display
+ *
+ * @since 200.8.0
+ */
+@Composable
+public fun OverviewMap(
+    viewpoint: Viewpoint?,
+    modifier: Modifier = Modifier,
+    scaleFactor: Double = 25.0,
+    symbol: Symbol = remember {
+        SimpleMarkerSymbol(
+            style = SimpleMarkerSymbolStyle.Cross,
+            color = Color.red,
+            size = 20.0f
+        )
+    },
+    arcGISMap: ArcGISMap = remember { ArcGISMap(BasemapStyle.ArcGISLightGray) }
+) {
+    OverviewMapImpl(
+        viewpoint = viewpoint,
+        arcGISMap = arcGISMap,
+        modifier = modifier,
+        symbol = symbol,
+        scaleFactor = scaleFactor,
+    )
+}
+
+/**
+ * Internal implementation of the OverViewMap.
+ *
+ * @param viewpoint the viewpoint of the main view this overview map is for, this be provided as a
+ * [ViewpointType.CenterAndScale]
+ * @param arcGISMap the map to display
+ * @param modifier the modifier to apply
+ * @param visibleArea the visible area. The visible area is only applicable when the main view is
+ * a [MapView] and should be null for a [SceneView].
+ * @param symbol the symbol used to visualize the main view's viewpoint. When the main view is a
+ * [MapView] the symbol should be suitable for a polygon geometry, and when the main view is a
+ * [SceneView] the symbol should be suitable for a point geometry.
+ * @param scaleFactor the scale factor applied to the viewpoint
+ * @since 200.8.0
+ */
+@Composable
+private fun OverviewMapImpl(
+    viewpoint: Viewpoint?,
+    arcGISMap: ArcGISMap,
+    modifier: Modifier = Modifier,
+    visibleArea: Polygon? = null,
+    symbol: Symbol? = null,
+    scaleFactor: Double = 25.0,
+) {
+    val proxy = remember {
+        MapViewProxy()
+    }
+
+    val graphic = remember {
+        Graphic()
+    }
+
+    val graphicsOverlay = remember {
+        GraphicsOverlay(listOf(graphic))
+    }
+
+    val graphicsOverlays = remember {
+        listOf(graphicsOverlay)
+    }
```

Comment:
Think it's fine to cut some of the whitespace here
```suggestion
    val proxy = remember { MapViewProxy() }
    
    val graphic = remember { Graphic() }
    val graphicsOverlay = remember { GraphicsOverlay(listOf(graphic)) }
    val graphicsOverlays = remember { listOf(graphicsOverlay) }
```